independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/commands/handlers/UpdateProjectCommandHandler.java

```diff
@@ -83,6 +83,8 @@ public QuarkusCommandOutcome execute(QuarkusCommandInvocation invocation) throws
                             QuarkusProjectHelper.artifactResolver(), updateRecipesVersion, request);
                     final String rewritePluginVersion = invocation.getValue(UpdateProject.REWRITE_PLUGIN_VERSION);
                     final boolean rewriteDryRun = invocation.getValue(UpdateProject.REWRITE_DRY_RUN, false);
+                    invocation.log().warn(
+                            "The project rewrite feature does not yet handle project extensions versions update. If needed, apply them manually.");
```

Comment:
Maybe this would be better?


```suggestion
                            "The update feature does not yet handle updates of the extension versions. If needed, update your extensions manually.");;
```